templates/post-edit.html

```diff
@@ -32,16 +29,23 @@
                                     {% endfor %}
                                 {% endif %}
                             </div>
-                            <div class=" container-fluid">
+                            <div class="container-fluid three-buttons">
```

Comment:
Why "three-buttons"?
What if we add more button in further steps? will we change the name as "four-buttons"
or
What if we have another three buttons in other templates? will you name them also "three-buttons"?
Use meaningful names. Names that give us the functionality of the button.

Reply:
updated changes made. thank you for comments

Reply:
Thank you too, so if it is ok mark this conversations as resolved.
Button is down here

Reply:
Done